packages/cloudflare/src/request.ts

```diff
@@ -15,7 +17,18 @@ import { addCloudResourceContext, addCultureContext, addRequest } from './scope-
 import { init } from './sdk';
 
 interface RequestHandlerWrapperOptions {
-  options: CloudflareOptions;
+  options: CloudflareOptions & {
+    /**
+     * Enable or disable the automatic continuation of traces from the propagation context.
+     *
+     * When enabled, the SDK will continue a trace from the propagation context if it is present.
+     *
+     * When disabled, the SDK will fall back to the default case of continuing a trace from the request headers if they are present.
+     *
+     * @default false
+     */
+    continueTraceFromPropagationContext?: boolean;
```

Comment:
@AbhiPrasad Is this a good place for adding the option? As it's only needed in the requestHandler, I added it only here and not in the general Cloudflare SDK options.